pkg/server/sessions/manager.go

```diff
@@ -65,30 +63,38 @@ func NewManager(options *Options) (*manager, error) {
 	if options == nil {
```

Comment:
just wondering if the input param could just be `Options`

Reply:
That could be inconsistent with other parts of the code (like other modules) and all those functions like `DefaultOptions`, `With...` modifiers - all those return by pointer.

We could of course change this to a functional-like approach - where we would work on values everywhere (like each `With...` call returns a new Options object with modifications applied instead of modifying the object itself). But then I think we should rather consistently change it in the overall codebase.